### PR TITLE
docs: fix typesetting

### PR DESCRIPTION
**Pull request by 1046102779**
https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#install-dapr-a-private-dapr-helm-chart

image

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md [hunk 1]
```diff
@@ -65,7 +65,9 @@ There are some scenarios where it's necessary to install Dapr from a private Hel
 - pulling Helm charts from trusted registries that are managed and maintained by your organization
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md:66]: Encasing lines 67-71 in a code block.

```suggestion
```bash
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md [hunk 2]
```diff
@@ -65,7 +65,9 @@ There are some scenarios where it's necessary to install Dapr from a private Hel
...
 
 export DAPR_HELM_REPO_URL="https://helm.custom-domain.com/dapr/dapr"
+
 export DAPR_HELM_REPO_USERNAME="username_xxx"
+
 export DAPR_HELM_REPO_PASSWORD="passwd_xxx"
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md:72]: Per comment above, add ``` here to close out code block.

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
quick edit for clarity, otherwise lgtm

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request closed without merging**